src/components/search/search.tsx

```diff
@@ -0,0 +1,16 @@
+import { AppButton } from 'components/app-button/app-button';
+import styles from './search.module.scss';
+
+const placeholder = 'What do you want to watch?';
+
+export const Search = () => {
+  const handleClick = () => {
+    console.log('search');
+  };
+  return (
```

Comment:
Advise to **always** add empty row before `return` statement, it is a good code practice. You can configure ESLint to add [this](https://eslint.org/docs/latest/rules/padding-line-between-statements) rule